bagit.py

```diff
@@ -784,6 +784,15 @@ def _make_tagmanifest_file(alg, bag_dir):
         for digest, filename in checksums:
             tagmanifest.write('%s %s\n' % (digest, filename))
 
+def _find_tag_files(bag_dir):
+    for dirName, subdirList, fileList in os.walk(bag_dir):
```

Comment:
PEP-8 encourages `dir_name`,  etc. I generally prefer to use plural names like `subdirectories`, `filenames`, rather than a `list` suffix but am ±0 on that.